packages/engine/src/interaction/systems/GrabbableSystem.tsx

```diff
@@ -390,19 +397,26 @@ const execute = () => {
 }
 
 const executeInput = () => {
-  const inputPointerEntity = InputPointerComponent.getPointerForCanvas(Engine.instance.viewerEntity)
-  if (!inputPointerEntity) return
+  // const inputPointerEntity = InputPointerComponent.getPointerForCanvas(Engine.instance.viewerEntity)
```

Comment:
i we arent planning on using spatial input here, we should remove the commented code

Reply:
done, got rid of this as it didn't make sense with interactable distance checking anymore, and also the use case of XR hands being basically on top of a grabbable but perhaps rotated wrong (not pointing at it) felt like a miss if it fails to grab